Jenkinsfile

```diff
@@ -23,6 +24,8 @@ def withGo(Closure body) {
             "GOPATH+WS=${env.WORKSPACE}",
             "PATH+GOBIN=${env.WORKSPACE}/bin",
             "HOME=${env.WORKSPACE}",
+            // https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html
+            "PATH+AWSIAMAUTHENTICATOR=${tool 'aws-iam-authenticator'}"
```

Comment:
I would suggest performing the tool installation only where it will be used. By defining it here, we're installing it every time the `withGo` function is called.

Reply:
Done.